### DISCUSSION TITLE
Можно ли каким либо образом заставить работать byedpi с приложением ютуба на андройде?

### DISCUSSION THREAD

**Discussion by himkalio** [hufrea/byedpi#50] [General]
как я понял, прога распространяется только на браузер, про само приложение ютуба никто ничего не говорит. если такое не представляется возможным, посоветуйте какие либо аналоги ютуба, "моды" какие либо. а то каждый раз не хочется врубать впн.

**Comment by lufog** [comment]
[ByeDPI for Android](https://github.com/dovecoteescapee/ByeDPIAndroid) VPN'ом притворяется (если я правильно помню), так что по идее и с приложениями должен работать.

Если устройство с root'ом, можно попробовать [DPI Tunnel](https://github.com/nomoresat/DPITunnel-android) в нем есть авто подбор методов обхода. Хоть он давно не обновляется, но работает вполне исправно, сам им пользуюсь.
